app/views/helpers/feed/update.phtml

```diff
@@ -48,7 +48,7 @@
 					<a class="btn" target="_blank" href="<?php echo $this->feed->url(); ?>"><?php echo _i('link'); ?></a>
 				</div>
 
-				<a class="btn" target="_blank" href="http://validator.w3.org/feed/check.cgi?url=<?php echo $this->feed->url(); ?>"><?php echo _t('sub.feed.validator'); ?></a>
+				<a class="btn" target="_blank" href="http://validator.w3.org/feed/check.cgi?url=<?php echo urlencode(str_replace('&amp;', '&', $this->feed->url())); ?>"><?php echo _t('sub.feed.validator'); ?></a>
```

Comment:
Thanks!
I think instead of `str_replace()`, it would be better to use:
```php
htmlspecialchars_decode(..., ENT_QUOTES);
```
Furthermore, regarding the handling of spaces, if think `rawurlencode()` should be used instead of `urlencode()`.